pkg/sqlite/migrationsPostgres/1_initial.up.sql

```diff
@@ -0,0 +1,475 @@
+CREATE COLLATION IF NOT EXISTS NATURAL_CI (provider = icu, locale = 'en@colNumeric=yes');
+CREATE TABLE blobs (
+    checksum varchar(255) NOT NULL PRIMARY KEY,
+    blob bytea
+);
+CREATE TABLE tags (
+  id serial not null primary key,
+  name varchar(255),
+  created_at timestamp not null,
+  updated_at timestamp not null, 
+  ignore_auto_tag boolean not null default FALSE, 
+  description text, 
+  image_blob varchar(255) 
+  REFERENCES blobs(checksum), 
+  favorite boolean not null default false
+);
+CREATE TABLE folders (
+  id serial not null primary key,
+  path varchar(255) NOT NULL,
```

Comment:
IMPORTANT: In this migration file, please use `text` everywhere you're using `varchar(255)`. I see that you're using `varchar(255)` in many places including things where it's very much realistic to have longer strings (URLs, file paths).

This doesn't matter with SQLite because [SQLite doesn't enforce the length anyways](https://www.sqlite.org/faq.html):

 > 9) What is the maximum size of a VARCHAR in SQLite?

 >    SQLite does not enforce the length of a VARCHAR. You can declare a VARCHAR(10) and SQLite will be happy to>  store a 500-million character string there. And it will keep all 500-million characters intact. Your content is>  never truncated. SQLite understands the column type of "VARCHAR(N)" to be the same as "TEXT", regardless of the>  value of N. 

However, Postgres pretty much does.

`text` is a lot better here than varchar in all cases

Reply:
Grrrr sqlite, grrr 😡
Yeah i'll change this

Reply:
If this surprises you, wait till you also read this: https://www.sqlite.org/faq.html#q3 :D